### ISSUE TITLE
combine_lipidData duplicates f_data

### ISSUE THREAD

**Issue by leac176** [pmartR/pmartR#175]
When combining lipid data, it duplicates the f_data leaving you with for example a method.x and a method.y column. Each contains the same information.

**Comment by stratkg** [comment]
Need to revisit this with Daniel...there was a reason that this happened...

**Comment by clabornd** [comment]
@stratkg @leac176 

Sry yes I will put this back on my radar.  For note taking purposes:  it is basically because, in an attempt to preserve f_data information, I combine the two f_data's and create de-duped column names.  Later, if they wanted to keep group info, I find which of those columns have the same values as the original grouping column and use those columns to re-run group_designation (same with covariates).  

New way per discussion with @lmbramer :  

- If user wants to keep group info, force both group_DF's to be present and for the groups/covariate structures to be functionally the same.
- full_join f_datas on sample name *and* all covariates/main effects with the same names and check that the join doesn't produce new rows. (other duplicate non-sampleID/covariate/main-effect columns will be copied)
- Re-run group_designation on the original main effects and covariates from the first object's group_DF.

**Comment by stratkg** [comment]
Closing this issue after #186 was resolved and updates merged into develop
